src/applications/caregivers/components/PreSubmitInfo/index.jsx

```diff
@@ -143,49 +158,52 @@ const PreSubmitCheckboxGroup = ({ onSectionComplete, formData, showError }) => {
         <PrivacyPolicy />
       </SignatureCheckbox>
 
-      <SignatureCheckbox
-        fullName={formData.primaryFullName}
-        label={primaryLabel}
-        signatures={signatures}
-        setSignature={setSignatures}
-        isRequired
-        showError={showError}
-      >
-        <h3 className="vads-u-margin-top--4">
-          Primary Family Caregiver applicant&apos;s statement of truth
-        </h3>
-
-        <p className="vads-u-margin-y--4">
-          I certify that I am at least 18 years of age.
-        </p>
-
-        <p>
-          I certify that I am a family member of the Veteran named in this
-          application or I reside with the Veteran, or will do so upon
-          designation as the Veteran's Primary Family Caregiver.
-        </p>
-
-        <p>
-          I agree to perform personal care services as the Primary Family
-          Caregiver for the Veteran named on this application.
-        </p>
-
-        <p>
-          I understand that the Veteran or Veteran’s surrogate may request my
-          discharge from the Program of Comprehensive Assistance for Family
-          Caregivers (PCAFC) at any time. I understand that my designation as a
-          Primary Family Caregiver may be revoked or I may be discharged from
-          the program by the Secretary of Veterans Affairs or his designee, as
-          set forth in 38 CFR 71.45.
-        </p>
-
-        <p>
-          I understand that participation in PCAFC does not create an employment
-          relationship between me and the Department of Veterans Affairs.
-        </p>
-
-        <PrivacyPolicy />
-      </SignatureCheckbox>
+      {hasPrimary && (
```

Comment:
only show If party is applicable

Reply:
Isn't a party _always_ available? 
<img alt="ghe-emoji" title=":_tuzki_dance_with_me:" src="https://cloud.githubusercontent.com/assets/136959/13203522/28de7908-d880-11e5-8715-1da2ba976252.gif">